src/axom/inlet/Inlet.hpp

```diff
@@ -249,13 +252,20 @@ class Inlet : public SchemaCreator
    */
   bool get(const std::string& name, std::string& value);
 
+  void registerDocWriter(std::shared_ptr<DocWriter> writer);
+
+  void writeDocs();
```

Comment:
I think this is the last request from me hopefully.  Please document your functions in doxygen format (basically how i have already done the rest of them)